logging/access.go

```diff
@@ -90,6 +97,14 @@ func (f *accessLogFormatter) Format(e *logrus.Entry) ([]byte, error) {
 	return []byte(fmt.Sprintf(f.format, values...)), nil
 }
 
+func stripQueryString(u string) string {
+	if i := strings.IndexRune(u, '?'); i < 0 {
```

Comment:
I think the anchor tag is browser functionality and should not be sent to the server. So I believe no, there's no such sane scenario. Can someone think about any?